mcs/class/corlib/Test/System/EnvironmentTest.cs

```diff
@@ -239,5 +239,11 @@ public void MachineNameIsNotFullyQualifiedDomainName ()
 			Assert.AreEqual (-1, Environment.MachineName.IndexOf("."));
 		}
 #endif
+
+		[Test] // https://github.com/mono/mono/issues/13030
+		public void GetLogicalDrivesNotEmpty ()
+		{
+			Assert.AreNotEqual (0, Directory.GetLogicalDrives ().Length);
```

Comment:
should be `Environment.GetLogicalDrives ()` here.

better would be to use `CollectionAssert.IsNotEmpty ()` so we get better message in case of failure.

Reply:
Oh, forgot about CollectionAssert thanks